### ISSUE TITLE
Yet another regex template parsing issue 

### ISSUE THREAD

**Issue by renanlecaro** [Akryum/vue-meteor#70]
Hey, thanks for the last fix ! 

I found another bug in the .vue file parsing
this works
`<button :disabled="1<2">Works</button>`
but this does not
`<button :disabled="2>1">Crashes</button>`

Using modules : 

```
akryum:vue-component@0.7.5
akryum:vue-component-dev-client@0.2.0
akryum:vue-component-dev-server@0.0.2
```

Also : 
![image](https://cloud.githubusercontent.com/assets/3457214/19146920/f7e7a934-8bb5-11e6-9c44-b3f1ebc22f2e.png)

**Comment by ykshev** [comment]
Not sure that I have the same issue, but
```p(v-if="true")``` or ```p(v-if="false")``` or  ```p(v-if="someComputedValue")``` — works, but ```p(v-if="2 > 1")``` — doesn't.
Meteor, vue 2.0.

Why it could happen?

**Comment by pyrostrex** [comment]
This happened because the current tagReg confused when there's more than one `>` characters.

http://haacked.com/archive/2004/10/25/usingregularexpressionstomatchhtml.aspx/

Sadly, the above link's regex only works on a single-line html element.

I tried lots of different regex but I can't seem to find one that works perfectly. Maybe it's just me.

I settled with a workaround of using a regex that match the first part of the html element and changed the tag replacement script to only replace rather than replace and assigning back the found attributes.

Not sure if it is good enough for PR.
